src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRow.test.tsx

```diff
@@ -15,6 +15,10 @@ describe("TransactionRow", () => {
 		wallet: () => ({
 			...TransactionFixture.wallet(),
 			currency: () => "DARK",
+			network: () => ({
+				coin: () => "DARK",
+				id: () => "ark.mainnet",
```

Comment:
although it will work, for consistency I'd prefer to use the right `id` when using it in combination with DARK. Applies to the other changes in this PR as well

```suggestion
			network: () => ({
				coin: () => "DARK",
				id: () => "ark.devnet",
```

Reply:
@patricio0312rev keywords: **Applies to the other changes in this PR as well**